source/includes/harvest/_introduction.md

```diff
@@ -166,6 +171,7 @@ The timestamps below are Eastern Time.
 
 | Date                    | Description                                                                                                    |
 | ----------------------- | -------------------------------------------------------------------------------------------------------------- |
+| May 7, 2019 3:30:00 PM  | Added [Demographic Data endpoints](#demographic-data)                                                          |
 | Apr 25, 2019 5:00:00PM  | Added ability to change prospect pool and stage via [PATCH: Update Application](#patch-update-application)     |
```

Comment:
@capablemonkey I think you might need to rebase. The prospect pool/stage doc card got shipped and rolled back, so it shouldn't be listed here (yet).

Reply:
https://developers.greenhouse.io/harvest.html#harvest-change-log

Reply:
fixed!